--2014-02-28 11:39:16--  http://stash.osgconnect.net/+sthapa/cvmfs-root.tar.gz
Resolving stash.osgconnect.net... 192.170.227.197
Connecting to stash.osgconnect.net|192.170.227.197|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://raw.stash.osgconnect.net/+sthapa/cvmfs-root.tar.gz [following]
--2014-02-28 11:39:16--  http://raw.stash.osgconnect.net/+sthapa/cvmfs-root.tar.gz
Resolving raw.stash.osgconnect.net... 192.170.227.197
Reusing existing connection to stash.osgconnect.net:80.
HTTP request sent, awaiting response... 200 OK
Length: 38883 (38K) [application/x-gzip]
Saving to: `cvmfs-root.tar.gz'

     0K .......... .......... .......... .......              100%  199M=0s

2014-02-28 11:39:16 (199 MB/s) - `cvmfs-root.tar.gz' saved [38883/38883]

/cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase/utilities/createUserASetup.sh: line 40: /.asetup: Permission denied
Warning in <TClass::TClass>: no dictionary for class pair<string,string> is available
140228 12:03:18 25421 Xrd: XrdClientMessage::ReadRaw: Failed to read header (8 bytes).
140228 12:05:53 25291 Xrd: ReadPartialAnswer: Failed to read msg from connmgr (server [faxbox.mwt2.org:1094]). Retrying ...
140228 12:05:54 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:05:54 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:05:54 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:05:54 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:05:59 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:05:59 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:05:59 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:05:59 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:04 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:04 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:04 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:04 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:09 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:09 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:09 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:09 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:14 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:14 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:14 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:14 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:19 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:19 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:19 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:19 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:24 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:24 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:24 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:24 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:29 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:29 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:29 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:29 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:34 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:34 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:34 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:34 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:39 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:39 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:39 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:39 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:44 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:44 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:44 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:44 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:49 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:49 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:49 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:49 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:54 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:54 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:54 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:54 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:06:59 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:06:59 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:06:59 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:06:59 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:07:04 25291 Xrd: HandleServerError: Communication error with server [faxbox.mwt2.org:1094]. Rebouncing here.
140228 12:07:04 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:07:04 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
140228 12:07:04 25291 Xrd: GoToAnotherServer: Error connecting to [faxbox.mwt2.org:1094
140228 12:07:09 25291 Xrd: SendGenCommand: Too many redirections for request  kXR_readv. Aborting command.
140228 12:07:09 25291 Xrd: GetParallelStreamCount: Unknown logical conn -1
140228 12:07:09 25291 Xrd: GetParallelStreamToUse: Unknown logical conn -1
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: SendGenCommand: Too many redirections for request  kXR_read. Aborting command.
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:2765170898, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=1, nerrors=1, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=2, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=3, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=4, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=5, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=6, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=7, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=8, basketnumber=57
Error in <TBranch::GetBasket>: File: root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212000.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid00994158_00/NTUP_SMWZ.00994158._000002.MWT2.root.1 at byte:0, branch:EF_2b35_loose_3j35_a4tchad_4L1J15, entry:21503, badread=0, nerrors=9, basketnumber=57
140228 12:07:09 25291 Xrd: GetParallelStreamCount: Unknown logical conn -1
140228 12:07:09 25291 Xrd: GetParallelStreamToUse: Unknown logical conn -1
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: SendGenCommand: Too many redirections for request  kXR_readv. Aborting command.
140228 12:07:09 25291 Xrd: GetParallelStreamCount: Unknown logical conn -1
140228 12:07:09 25291 Xrd: GetParallelStreamToUse: Unknown logical conn -1
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: SendGenCommand: Too many redirections for request  kXR_read. Aborting command.
140228 12:07:09 25291 Xrd: WriteToServer: Unknown logical conn -1
140228 12:07:09 25291 Xrd: SendGenCommand: Too many redirections for request  kXR_close. Aborting command.
140228 12:07:09 25291 Xrd: Connect: can't open connection to [faxbox.mwt2.org:1094]
140228 12:07:09 25291 Xrd: XrdNetFile: Error creating logical connection to faxbox.mwt2.org:1094
Error in <TXNetFile::CreateXClient>: open attempt failed on root://faxbox.usatlas.org//user/sthapa/FDR/user.flegger.MWT2.data12_8TeV.00212199.physics_Muons.merge.NTUP_SMWZ.f479_m1228_p1067_p1141_tid01007410_00/NTUP_SMWZ.01007410._000053.MWT2.root.1

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================

Thread 2 (Thread 0x2b2e1b558700 (LWP 25413)):
#0  0x0000003f384accdd in nanosleep () from /lib64/libc.so.6
#1  0x0000003f384acb50 in sleep () from /lib64/libc.so.6
#2  0x00002b2e1b11ee3c in GarbageCollectorThread (arg=0xde3010, thr=<value optimized out>) at /afs/slac/public/software/scalla/releases/v3.2.7/xrootd-3.2.7/src/XrdClient/XrdClientConnMgr.cc:73
#3  0x00002b2e1aea2b0f in XrdSysThread_Xeq (myargs=0x783540) at /afs/slac/public/software/scalla/releases/v3.2.7/xrootd-3.2.7/src/XrdSys/XrdSysPthread.cc:67
#4  0x0000003f388079d1 in start_thread () from /lib64/libpthread.so.0
#5  0x0000003f384e8b6d in clone () from /lib64/libc.so.6

Thread 1 (Thread 0x2b2e1a823080 (LWP 25291)):
#0  0x0000003f384ac8fd in waitpid () from /lib64/libc.so.6
#1  0x0000003f3843e909 in do_system () from /lib64/libc.so.6
#2  0x0000003f3843ec40 in system () from /lib64/libc.so.6
#3  0x00002b2e155287cc in TUnixSystem::StackTrace() () from /cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase/x86_64/root/5.34.14-x86_64-slc6-gcc4.7/lib/libCore.so
#4  0x00002b2e1552b003 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase/x86_64/root/5.34.14-x86_64-slc6-gcc4.7/lib/libCore.so
#5  <signal handler called>
#6  0x000000000040420e in readFile(std::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::basic_string<char, std::char_traits<char>, std::allocator<char> >, float, float, std::basic_string<char, std::char_traits<char>, std::allocator<char> >, rootStats*) ()
#7  0x0000000000404c1a in main ()
===========================================================


The lines below might hint at the cause of the crash.
If they do not help you then please submit a bug report at
http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#6  0x000000000040420e in readFile(std::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::basic_string<char, std::char_traits<char>, std::allocator<char> >, float, float, std::basic_string<char, std::char_traits<char>, std::allocator<char> >, rootStats*) ()
#7  0x0000000000404c1a in main ()
===========================================================



real	27m54.031s
user	1m16.857s
sys	0m6.583s
